src/theory/quantifiers/sygus/sygus_unif_rl.h

```diff
@@ -231,6 +244,8 @@ class SygusUnifRl : public SygusUnif
                        const std::vector<Node>& conds);
 
    private:
+    /** built solutions */
```

Comment:
Can you document this a bit more?  When is this added to? What solutions do you mean?